tensorflow/tools/ci_build/Dockerfile.gpu

```diff
@@ -1,7 +1,12 @@
-FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu14.04
```

Comment:
Exclude this file? It won't build for the reasons I mentioned in the other change.